apps/web/components/trials/Results.tsx

```diff
@@ -152,7 +152,8 @@ const Results = (props: {trials: [], getTrialsForUsersInGroupLoading: boolean})
         trialRetCount: (cur.trialStatus === TrialStatusEnum[TrialStatusEnum.PENDING]) ? '' : cur.trialRetCount,
         matchedDate: cur.matchedDate? cur.matchedDate : '',
         trialStatus: cur.trialStatus,
-        short_title: cur.short_title
+        short_title: cur.short_title,
+        matchSentDate: cur.matchSentDate
```

Comment:
What does it display if it's not defined? Do we need to do the same logic for matchedDate to check if it's defined?